ProcessLib/SmallDeformationWithLIE/Common/FractureProperty.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <Eigen/Eigen>
+
+#include "MeshLib/Elements/Element.h"
+#include "MeshLib/Node.h"
+
+#include "ProcessLib/Parameter/Parameter.h"
+
+#include "FractureProperty.h"
+#include "Utils.h"
+
+
+namespace ProcessLib
+{
+namespace SmallDeformationWithLIE
+{
+
+struct FractureProperty
+{
+    int mat_id = 0;
+    Eigen::Vector3d point_on_fracture;
+    Eigen::Vector3d normal_vector;
+    /// Rotation matrix from global to local coordinates
+    Eigen::MatrixXd R;
+    /// Initial aperture
+    ProcessLib::Parameter<double>* aperture0 = nullptr;
+};
+
+/// configure fracture property based on a fracture element
+inline void setFractureProperty(unsigned dim, MeshLib::Element const& e,
+                                FractureProperty &frac_prop)
+{
+    for (int j=0; j<3; j++)
+        frac_prop.point_on_fracture[j] = e.getNode(0)->getCoords()[j];
```

Comment:
As it assumes a fracture forms a straight line, it doesn't matter whether I take the first or second node. I'll add a comment there.